migrations/apply_click_coordinates.py

```
@@ -0,0 +1,34 @@
+
+    db_url = os.getenv("DATABASE_URL")
+    if not db_url:
+        print("Missing DATABASE_URL")
+        exit(1)
+        
+    print("Applying 057_click_coordinates migration...")
+    conn = await asyncpg.connect(db_url)
+    try:
+        await conn.execute(sql)
+        print("Migration applied successfully via asyncpg direct connection!")
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)